rosetta-source/src/main/rosetta/mapping-datamodel-type.rosetta

```diff
@@ -0,0 +1,2 @@
+namespace template.mapping.datamodel
```

Comment:
Let's use

instead of `datamodel`

we can use the templated string:

`templateDataModel`

so for Monopoly: we should use the full namespace : `monopoly.mapping.monopolyModel`

Reply:
Or maybe we can parameterise the last bit (`monopolyModel`) so we can control it?